examples/gradle-v2/custom-format-example/README.md

```diff
@@ -0,0 +1,17 @@
+## Dokka custom format example
```

Comment:
It would be better to rename this example (folder and title) to something like `custom-styles` to not confuse with Dokka formats

Reply:
YES this same confusing has been bugging me for a while